src/Api.Rest.Dtos/JsonConverters/JsonDataValueConverter.cs

```diff
@@ -0,0 +1,45 @@
+﻿#region copyright
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+/* Carl Zeiss IMT (IZM Dresden)                    */
+/* Softwaresystem PiWeb                            */
+/* (c) Carl Zeiss 2022                             */
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#endregion
+
+namespace Zeiss.PiWeb.Api.Rest.Dtos.JsonConverters
+{
+	#region usings
+
+	using System;
+	using System.Text.Json;
+	using System.Text.Json.Serialization;
+	using Zeiss.PiWeb.Api.Rest.Dtos.Data;
+
+	#endregion
+
+	/// <summary>
+	/// Specialized <see cref="JsonConverter"/> for <see cref="DataValueDto"/>-objects.
+	/// </summary>
+	public sealed class JsonDataValueConverter : JsonConverter<DataValueDto>
```

Comment:
What came to my mind right now: we do have to keep an eye on the compatibility regarding deserialization. I think simply removing the DataCharacteristic would break all new clients working with an older server, since they no longer can deserialize the json. And new client with older server is quite often the case. The other way around we have the same problem. Is it possible to write a converter that can handle the old DataCharacteristic and map it to the new Guid-DataValueDto tuple and vice versa? Then we need to keep this compatibility code for some releases and remove it later.